server/notebooks/views.py

```diff
@@ -31,13 +31,16 @@ def notebook_view(request, pk):
     notebook = get_object_or_404(Notebook, pk=pk)
     if "revision" in request.GET:
         revision = get_object_or_404(NotebookRevision, pk=int(request.GET["revision"]))
```

Comment:
I don't know if there's any fear but if this can't be turned into an `int` this could 500 the API.

Reply:
Also, do you want to make sure this is revision is assigned to the correct `Notebook` also? I.e. `get_object_or_404(NotebookRevision, notebook=notebook, pk=int(request.GET["revision"]))`

Reply:
both of these are good points, will fix before landing.